inventory-api-compose.yaml

```diff
@@ -25,3 +25,5 @@ storage:
     user: "postgres"
     password: "yPsw5e6ab4bvAGe5H"
     dbname: "spicedb"
+log:
+  level: "info"
```

Comment:
Can you also add `log: level` to `kafka-inventory-api.yaml` or `sso-inventory-api.yaml`